improver/utilities/spatial.py

```diff
@@ -302,43 +302,49 @@ def __init__(self, regrid=False):
         self.regrid = regrid
 
     @staticmethod
-    def _update_metadata(gradient, cube):
-        """Rename cube and remove inappropriate metadata.
+    def _update_metadata(gradient, cube, diff, axis):
+        """Rename cube and assign appropriate units.
 
         Args:
             gradient (iris.cube.Cube)
+                Cube containing gradients along the x or y axis.
             cube (iris.cube.Cube)
+                Cube used to calculate differences and gradients.
+            diff (iris.cube.Cube)
+                Cube containing differences along the x or y axis.
+            axis (str)
```

Comment:
It appears that the colons are not necessary for sphinx, just the new line and indent, so technically these are fine (though inconsistent with the rest of the file). Up to you if you want to add them.

Reply:
I've added colons for consistency.